projects/libvpx/README.md

```diff
@@ -0,0 +1,52 @@
+# Submit a Patch to oss-fuzz repo
```

Comment:
I added this README for libaom, but it is pretty generic I think.
If useful, maybe it should be moved to a common location in oss-fuzz repo  @oliverchang ?

Reply:
For the rest of the code we could pull it into libvpx. It's been helpful to get feedback from the oss-fuzz side when doing libwebp, libaom, but we can always cc you on gerrit. We had plans to move libwebp to upstream, but it's been a low priority since things are working here. libaom should follow suit too.

Reply:
ccing us on gerrit when you're making fuzzer changes SGTM. we prefer to keep this repo clean and contain minimal code/build configuration.

Reply:
James, In order to move this to libvpx and ensure it builds there,
we would need to do either limit this code to be built only when using recent clang (which supports -fsanize=fuzzer) or build libFuzzer.a from source. Second approach would require adding third_party libFuzzer repo.
Also, would this need to be built as part of configure and make setup or would a simple Makefile be enough?

Reply:
Requiring a recent clang is fine, libvpx's build is run through 6.0.1 currently.
My preference would be to have it as a separate tool target in configure so we could pick up the same flags used by the library for warnings, etc. As a first pass the code with instructions at the top of the file will be enough I think.

Reply:
Thanks! I'll merge this.